package.json

```diff
@@ -19,6 +19,7 @@
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
     "@hookstate/core": "^4.0.0-rc21",
+    "@ipfs-shipyard/ignite-metrics": "https://github.com/ipfs-shipyard/ignite-metrics#feat/useStorageForBrowsers-with-dist",
```

Comment:
will update when https://github.com/ipfs-shipyard/ignite-metrics/pull/64 is merged and new ignite-metrics is released